test/exsom_test.exs

```diff
@@ -5,7 +5,7 @@ defmodule ExsomTest do
   @test_xml ~S[<foo attr="yo"><bar>1</bar></foo>]
 
 
-  test "integration / model -> instance -> write (file)" do
+  test "integration / model (file) -> instance -> write" do
```

Comment:
Okay, cool. In that case, it may be better to name the test for the goal that we're trying to achieve, rather than a sketch of technical details to achieve it. Thoughts?